src/sql/src/plan/statement.rs

```diff
@@ -480,7 +480,7 @@ fn plan_index_options(options: &[SqlOption]) -> Result<Vec<IndexOption>, anyhow:
         Some(Value::String(window)) => {
             let window = match window.as_str() {
                 "off" => None,
-                s => Some(parse_duration::parse(s)?),
+                s => Some(strconv::parse_interval(s)?.duration()?),
```

Comment:
any particular reason you didn't use `repr::util::parse_duration` here?

Reply:
🤦‍♂️ thanks. This is the call site that _inspired_ that helper method and I completely forgot to use it here.